### PR TITLE
awesome.md

### PR DESCRIPTION
**Pull request by abhigunj**

### CODE REVIEW

File: docs/awesome.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
+# How community is using awesome OpenXLA?
```
Comment (GleasonK) [inline docs/awesome.md:1]:
```
# OpenXLA Awesome

How is the community using OpenXLA? This page consolidates links to repos and projects using OpenXLA to provide inspiration and code pointers!

Have a project that uses OpenXLA? Send us a PR and add it to this page!
```

File: docs/awesome.md [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+## ML Frameworks
+
+**JAX:** [uses OpenXLA]((https://jax.readthedocs.io/en/latest/quickstart.html)) as
```
Comment (GleasonK) [inline docs/awesome.md:5]:
```
### JAX

A machine learning framework with a numpy-like API for writing high performance 
ML models. JAX lowers to StableHLO, PJRT, and XLA for high performance
compilation.
```

File: docs/awesome.md [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+ StableHLO Transformations are a set of MLIR passes designed to optimize and
+ transform programs in the StableHLO dialect.
+
+## Tooling and Vizualization
+
+**Model Explorer:** Model Explorer can visualize StableHLO representations of models, providing
+insights into the compilation process within OpenXLA.
+
+**SHerLOC:** SHerLOC (StableHLO Rule Optimization and Composition) is a framework for
```
Comment (GleasonK) [inline docs/awesome.md:37]: Remove from v0, 3p community should opt in to having their project featured

File: docs/awesome_openxla.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
+# Awesome OpenXLA
```
Comment (GleasonK) [inline docs/awesome_openxla.md:1]: Can you also add a link in `_toc.yaml`?

Comment (abhigunj) [reply docs/awesome_openxla.md:1]: sg, adding under "title: Getting started"

File: docs/awesome_openxla.md [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+
+**How is the community using OpenXLA?** This page consolidates links to
```
Comment (GleasonK) [inline docs/awesome_openxla.md:3]: Just learned of JAX's awesome list today: https://github.com/n2cholas/awesome-jax

Let's remodel to match that, images didn't fit as well as I thought they would:


```
# Awesome OpenXLA

<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://raw.githubusercontent.com/openxla/xla/refs/heads/main/docs/images/openxla_dark.svg" class="devsite-dark-theme">
  <img alt="OpenXLA Ecosystem" src="https://raw.githubusercontent.com/openxla/xla/refs/heads/main/docs/images/openxla.svg">
</picture>

[OpenXLA](https://openxla.org) is open ecosystem of performant, portable, and extensible machine learning (ML) infrastructure components that simplify ML development by defragmenting the tools between frontend frameworks and hardware backends. Built by industry leaders in AI modeling, software, and hardware.

**How is the community using OpenXLA?** This page consolidates links to
repositories and projects using OpenXLA to provide inspiration and code pointers!

**Have a project that uses OpenXLA?** Send us a pull request and add it to this page!

## Frameworks

- [JAX](https://github.com/jax-ml/jax) is a ML framework with a
NumPy-like API for writing high-performance ML models <img src="https://img.shields.io/github/stars/jax-ml/jax?style=social" align="center">
- [PyTorch/XLA](https://github.com/pytorch/xla/) provides a bridge from PyTorch to OpenXLA and StableHLO <img src="https://img.shields.io/github/stars/pytorch/xla?style=social" align="center">
- [TensorFlow](https://github.com/tensorflow/tensorflow) is a long-standing ML framework with a large ecosystem <img src="https://img.shields.io/github/stars/tensorflow/tensorflow?style=social" align="center">

## PJRT Plugins

- [libTPU](https://cloud.google.com/tpu/docs/runtimes) allows models to execute on Google's Cloud TPUs

## Edge Compilation

- [Google AI Edge](https://ai.google.dev/edge) uses StableHLO as an input format to deploy to mobile devices using [LiteRT](https://ai.google.dev/edge/litert)
  + [AI Edge Torch](https://github.com/google-ai-edge/ai-edge-torch) exports PyTorch models for mobile deployment via StableHLO <img src="https://img.shields.io/github/stars/google-ai-edge/ai-edge-torch?style=social" align="center">

## Tooling and Visualization

- [Model Explorer](https://github.com/google-ai-edge/model-explorer) offers heirarchical graph visualization with support for StableHLO models <img src="https://img.shields.io/github/stars/google-ai-edge/model-explorer?style=social" align="center">
```

**Review by GleasonK — APPROVED** [review]

File: docs/awesome_openxla.md (hunk 5 above)
Comment (abhigunj) [reply docs/awesome_openxla.md]: updated.

**Review by GleasonK — APPROVED** [review]

**Pull request merged**